source/fundamentals/connection.txt

```diff
@@ -101,8 +101,8 @@ URI to specify the behavior of the client.
    * - **validateOptions**
      - boolean
      - ``false``
-     - Specifies whether to error when an unknown or incorrect option is
-       provided. If ``false``, the driver produces warnings only.
+     - Specifies whether to error when it is passed an unknown or incorrect
```

Comment:
How about: 
```suggestion
     - Specifies whether to error when passed an unknown or incorrect
```

Reply:
Thanks to both of you for identifying the ambiguity of this sentence. I am going to reword this to specify the object to which an incorrect param is passed:

> Specifies whether to error when the method parameters contain an unknown or incorrect option.